packages-native/intro-screen/src/IntroScreen.tsx

```diff
@@ -0,0 +1,98 @@
+        if (props.onDone && props.onDone.canExecute) {
+            props.onDone.execute();
+        }
+        setVisible(false);
+    }, []);
+    const onSlideChange = useCallback((next: number, previous: number) => {
+        if (props.onSlideChange && props.onSlideChange.canExecute) {
+            props.onSlideChange.execute();
+        }
+        console.warn(next, previous);
+    }, []);
+    const onSkip = useCallback(() => {
+        if (props.onSkip && props.onSkip.canExecute) {
+            props.onSkip.execute();
+        }
+        setVisible(false);
+    }, []);
+
+    const renderText = (caption?: DynamicValue<string>): ReactNode => {
+        if (caption && caption.status === ValueStatus.Available && caption.value) {
+            return <Text style={[styles.buttonText, { marginLeft: 5 }]}>{caption.value}</Text>;
+        }
+        return undefined;
+    };
+
+    const renderButton = (
+        property: string,
+        style: ViewStyle,
+        icon?: DynamicValue<NativeIcon>,
+        caption?: DynamicValue<string>
+    ): any => {
+        const returnObject = {} as any;
+        if (!icon || !icon.value) {
+            return returnObject;
+        }
+
+        returnObject[property] = () => (
+            <View style={[{ flexDirection: "row", justifyContent: "center" }, style]}>
+                <Icon icon={icon!.value} color={styles.buttonText.color ? styles.buttonText.color : "white"} />
+                {renderText(caption)}
+            </View>
+        );
+
+        return returnObject;
+    };
+
+    const renderLabel = (label?: DynamicValue<string>, defaultValue?: string): string => {
+        if (label && label.value) {
+            return label.value;
+        }
+        return defaultValue!;
+    };
+
+    const showSkipPrevious = props.buttonPattern === "all";
+    const showNextDone = props.buttonPattern !== "none";
+
+    return (
+        <Modal visible={visible} transparent>
+            <View style={props.showMode === "fullscreen" ? styles.fullscreenContainer : styles.cardContainer}>
+                <SwipeableContainer
+                    slides={props.slides}
+                    onDone={onDone}
+                    onSlideChange={onSlideChange}
+                    onSkip={onSkip}
+                    bottomButton={props.slideIndicators !== "between"}
+                    numberOfButtons={props.buttonPattern === "next_done" ? 1 : 2}
+                    showSkipButton={showSkipPrevious}
+                    showNextButton={showNextDone}
+                    showPrevButton={showSkipPrevious}
+                    showDoneButton={showNextDone}
+                    hidePagination={props.slideIndicators === "never"}
+                    skipLabel={renderLabel(props.skipCaption, "Skip")}
+                    prevLabel={renderLabel(props.previousCaption, "Prev")}
+                    nextLabel={renderLabel(props.nextCaption, "Next")}
+                    doneLabel={renderLabel(props.doneCaption, "Done")}
```

Comment:
Default should should not have hardcode language, let go for no text, only  icons, till we have somethings like system text for widgets.